modifications/microshift-rebase

```diff
@@ -15,28 +15,24 @@ fail() {
     exit 1
 }
 
+# If MICROSHIFT_NO_REBASE is set and non-zero, stop execution here
 if [ -n "$MICROSHIFT_NO_REBASE" ] && [ "$MICROSHIFT_NO_REBASE" -ne 0 ] ; then
     echo "MICROSHIFT_NO_REBASE is defined. Rebase will not be run."
     exit 0
 fi
 
-# FIXME: rebase.sh requires Go yq (https://github.com/mikefarah/yq) instead of Python yq (https://github.com/kislyuk/yq),
-# which is used in the rest of the pipeline.
-# We install Go yq to /opt/yq-go/bin/ on buildvm to avoid conflict.
-# Use Go 1.19.5 as the version installed on buildvm (1.17) is no longer supported.
-# In the future we might have to run rebase.sh inside of a container to use specific versions of dependencies (including golang).
-export PATH=/opt/go-1.19.5/bin:/opt/yq-go/bin:$PATH
-
-if [ -n "$RELEASE_NAME" ] && [ -n "$LVMS_OPERATOR_BUNDLE" ]; then
-    # rebase against a named release
-    ./scripts/auto-rebase/rebase.sh to "$RELEASE_REPO":"$RELEASE_NAME"-x86_64 "$RELEASE_REPO":"$RELEASE_NAME"-aarch64 "$LVMS_OPERATOR_BUNDLE"
-    exit 0
-fi
-
-if [ -n "$MICROSHIFT_PAYLOAD_X86_64" ] && [ -n "$MICROSHIFT_PAYLOAD_AARCH64" ] && [ -n "$LVMS_OPERATOR_BUNDLE" ]; then
-    # rebase against specified release payloads
-    ./scripts/auto-rebase/rebase.sh to "$MICROSHIFT_PAYLOAD_X86_64" "$MICROSHIFT_PAYLOAD_AARCH64" "$LVMS_OPERATOR_BUNDLE"
-    exit 0
-fi
-
-fail "Environment variable RELEASE_NAME or LVMS_OPERATOR_BUNDLE is not set."
+DIR=$(dirname $(readlink -f $0))
+PODMAN_BUILD_CMD="podman build -t microshift-rebase-$BREW_TAG $DIR"
+echo $PODMAN_BUILD_CMD
+eval $PODMAN_BUILD_CMD
+
+# Run the podman container
+PODMAN_RUN_CMD="podman run --rm \
+-e RELEASE_NAME \
+-e LVMS_OPERATOR_BUNDLE \
+-e MICROSHIFT_PAYLOAD_X86_64 \
+-e MICROSHIFT_PAYLOAD_AARCH64 \
+-v $PWD:/workdir/microshift \
+microshift-rebase-$BREW_TAG"
+echo $PODMAN_RUN_CMD
```

Comment:
I saw you removed the invocation of ./scripts/auto-rebase/rebase.sh from Dockerfile but didn't add it back to `microshift-rebase`. Could you have a look at it again?

Reply:
Ah I am sorry I misunderstood. Doing so right now